python/ccxt/base/types.py

```diff
@@ -43,3 +43,18 @@ class Balance(TypedDict):
 
 
 IndexType = Union[str, int]
+
+class Trade(TypedDict):
```

Comment:
@carlosmiei did I miss smth here?

Reply:
@ttodua not sure, if the user changes the `number` format, some values like `price/cost/amount` would be string and not float

Reply:
ok changed type to none

Reply:
@ttodua hum, u sure `None` is ok? `Shouldn't we folloe Balance's approach? `Union[None, str, float]`?

Reply:
ah, missed about union. pushed update.

Reply:
build passed atm.